docs/development_kotlin_scripts.md

```diff
@@ -0,0 +1,15 @@
+# Development kotlin scripts hints
```

Comment:
can we perhaps have more detailed hints or docs, this seems incomplete.
WDYT?

Reply:
Yeah, I will add quickly tutorial on how to write and run standalone kotlin scripts.
Good point!

Reply:
We decided, to make scripts be part of flankScripts, so additional config is not needed